setup.py

```diff
@@ -51,7 +51,7 @@
     include_package_data=True,
     install_requires=[
         "eth-hash>=0.1.0,<1.0.0",
-        "eth-typing>=2.0.0,<3.0.0",
+        "eth-typing>=1.0.0,<3.0.0",
```

Comment:
I think this should be `>=1.3.0,<3.0.0` based on a quick search for what the previous constraint was.